benchmarks/geometry_providers.jl

```diff
@@ -0,0 +1,181 @@
+#=
+# Geometry providers
+
+This file benchmarks GeometryOps methods on every GeoInterface.jl implementation we can find, in order to test:
+a. genericness, i.e., does GeometryOps work correctly with all GeoInterface.jl implementations?
+b. performance, i.e., how does GeometryOps compare to the native implementation?
+c. performance issues in the packages' implementations of GeoInterface
+=#
+
+# First, we import the providers:
+using ArchGDAL, LibGEOS, Shapefile, GeoJSON, WellKnownGeometry, GeometryBasics, GeoInterface
+
+import GeometryOps as GO, GeoInterface as GI
+
+using BenchmarkTools, Chairmarks, CairoMakie, MakieThemes, DataFrames, Proj
+
+PROVIDERS = (ArchGDAL, LibGEOS, GeometryBasics, GI.Wrappers)
+
+
+import Polylabel
+
+water1 = ArchGDAL.fromWKT([readchomp(joinpath(dirname(dirname(pathof(Polylabel))), "test", "data", "water1.wkt")) |> String])
+water2 = ArchGDAL.fromWKT([readchomp(joinpath(dirname(dirname(pathof(Polylabel))), "test", "data", "water2.wkt")) |> String])
+
+water1_1 = GO.fix(water1, corrections = [GO.GEOSCorrection()])
```

Comment:
Should this just be `water1`? Given that `water2` wasn't renamed and `water1_1` isn't used again?

Reply:
I just ran this and realised that GEOS is messing this up...again.  I might just use it raw for now.